packages/app/src/sections/leaderboard/AllTime.tsx

```diff
@@ -20,10 +19,57 @@ import media from 'styles/media'
 import { getMedal } from 'utils/competition'
 import { staticMainnetProvider } from 'utils/network'
 
+type TraderCellProps = {
+	onClickTrader: (trader: string) => void
+	compact?: boolean
+	rank: number
+	traderEns?: string | null
+	traderShort: string
+	trader: string
+}
+
+const TraderCell: FC<TraderCellProps> = ({
+	onClickTrader,
+	compact,
+	rank,
+	traderEns,
+	traderShort,
+	trader,
+}) => {
+	const avatar = useENSAvatar(staticMainnetProvider, traderEns)
+
+	return (
+		<StyledOrderType onClick={() => onClickTrader(trader)}>
+			{compact && rank + '. '}
+			<StyledTrader>
```

Comment:
Could we use the other ENS Component here?

Reply:
Sure. Added.